spec/lib/avalon/batch/entry_spec.rb

```diff
@@ -23,6 +23,15 @@
   let(:entry_opts) { {user_key: 'archivist1@example.org', collection: collection} }
   let(:entry) { Avalon::Batch::Entry.new(entry_fields, entry_files, entry_opts, nil, nil) }
 
```

Comment:
Avalon seems to have issues with the `queue_adaptor` -- if you set it to `:test` for the entire suite, a bunch of tests fail. If you set it to `inline` (the default) a bunch of different test fail (fewer of them though). Ideally the default should be `:test`, but instead, there is some queue adaptor switching going on in the test suite. The upstream coders are aware of this issue.